pkg/configmap/services/services.go

```diff
@@ -0,0 +1,21 @@
+package services
+
+import (
+	"fmt"
+
+	"github.com/openshift/windows-machine-config-operator/version"
+)
+
+// ServicesConfigMapName is the name of the ConfigMap detailing service configuration for a specific WMCO version
+var ServicesConfigMapName string
+
+// init runs once, initializing global variables
+func init() {
+	ServicesConfigMapName = servicesConfigMapName()
+}
+
+// servicesConfigMapName returns the ConfigMap with the naming scheme:
+// windows-services-<MajorVersion>-<MinorVersion>-<PatchVersion>-<CommitHash>
```

Comment:
```suggestion
// windows-services-<WMCO Full Version>
```


The WMCO full version schema is maintained [elsewhere](https://github.com/openshift/windows-machine-config-operator/blob/master/build/build.sh#L11). Here, we just consume it.